e2e: settlements fail

--- FAIL: TestConsumerConnectsToProvider/Provider_settlement_flow (100.09s)
        connection_test.go:119: 
            	Error Trace:	connection_test.go:119
            	Error:      	Received unexpected error:
            	            	Post http://myst-provider:4050/transactor/settle/sync: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
            	Test:       	TestConsumerConnectsToProvider/Provider_settlement_flow
        connection_test.go:132: 
            	Error Trace:	connection_test.go:132
            	Error:      	Max difference between 689999692 and 690000000 allowed is 2, but difference was -308
            	Test:       	TestConsumerConnectsToProvider/Provider_settlement_flow

--- a/e2e/blockchain/deployer.go
+++ b/e2e/blockchain/deployer.go
@@ -144,7 +144,7 @@ func deployPaymentsv2Contracts(transactor *bind.TransactOpts, client *ethclient.
 	checkTxStatus(client, tx)
 
 	transactor.Nonce = lookupLastNonce(transactor.From, client)
-	tx, err = ts.Approve(transactor, registryAddress, big.NewInt(100000000000))
+	tx, err = ts.Approve(transactor, registryAddress, big.NewInt(100000000000000))
 	checkError("allow myst", err)
 	checkTxStatus(client, tx)
 
@@ -155,7 +155,7 @@ func deployPaymentsv2Contracts(transactor *bind.TransactOpts, client *ethclient.
 	tx, err = rt.RegisterAccountant(
 		transactor,
 		transactor.From,
-		big.NewInt(100000000000),
+		big.NewInt(100000000000000),
 		400,
 		big.NewInt(125000000000),
 	)

